operator/cmd/mesh/profile-common.go

```diff
@@ -244,8 +244,8 @@ func rewriteURLToLocalInstallPath(installPackagePath, profileOrPath string, skip
 // getProfileYAML returns the YAML for the given profile name, using the given profileOrPath string, which may be either
 // a profile label or a file path.
 func getProfileYAML(installPackagePath, profileOrPath string) (string, error) {
-	// If charts are a file path and profile is a name like default, transform it to the file path.
-	if installPackagePath != "" && helm.IsBuiltinProfileName(profileOrPath) {
+	// If charts are a file path and profile is not build in, transform it to the file path.
+	if installPackagePath != "" && !helm.IsBuiltinProfileName(profileOrPath) {
```

Comment:
I agree that there's a bug but I don't see how this would fix it. From a logic standpoint this change doesn't seem correct, can you provide more context on why you believe this works?

Reply:
Sorry, I found the mistake. It should be **not build in** and `profileOrPath` is **not file path**.
Is that right?

Of course we can ignore this error, but user have to put a profile on the path `{installPackagePath}/profiles/{profileOrPath}.yaml`

Reply:
I added a note in the original issue. https://github.com/istio/istio/issues/22368